src/views/Dashboard/components/AssetsList.tsx

```diff
@@ -0,0 +1,229 @@
+import useTreasury from "hooks/useTreasury";
+import { useWallet } from "use-wallet";
+
+const AssetsList: React.FC = () => {
+  const { totalYUsdValue, totalDPIValue, totalWETHValue, totalUMAValue, totalBalanceIndexCoop } = useTreasury();
+  const [yusdPrice, setYUSDPrice] = useState<number>();
+  const [dpiPrice, setDPIPrice] = useState<number>();
+  const [wethPrice, setWETHPrice] = useState<number>();
+  const [umaPrice, setUMAPrice] = useState<number>();
+  const [indexCoopPrice, setIndexCoopPrice] = useState<number>();
+  const [change24WETH, setChange24WETH] = useState<string>();
+  const [change24DPI, setChange24DPI] = useState<string>();
+  const [change24UMA, setChange24UMA] = useState<string>();
+  const [change24YUSD, setChange24YUSD] = useState<string>();
+  const [change24IndexCoop, setChange24IndexCoop] = useState<string>();
+  const { status } = useWallet();
+
+  const fetchOnce = useCallback(async () => {
+    const wethValues = await getValue("weth");
+    const dpiValues = await getValue("defipulse-index");
+    const umaValues = await getValue("uma");
+    const yusdValues = await getValue("yvault-lp-ycurve");
+    const indexCoopValues = await getValue("index-cooperative");
+
+    const yusdPrice = await getYUSDPrice();
+    const dpiPrice = await getDPIPrice();
+    const wethPrice = await getWETHPrice();
+    const umaPrice = await getUMAPrice();
+    const indexCoopPrice = await getINDEXCOOPPrice();
+    setChange24WETH(numeral(wethValues.market_data.price_change_percentage_24h_in_currency.usd).format("0.00a") + "%");
+    setChange24DPI(numeral(dpiValues.market_data.price_change_percentage_24h_in_currency.usd).format("0.00a") + "%");
+    setChange24UMA(numeral(umaValues.market_data.price_change_percentage_24h_in_currency.usd).format("0.00a") + "%");
+    setChange24YUSD(numeral(yusdValues.market_data.price_change_percentage_24h_in_currency.usd).format("0.00a") + "%");
+    setChange24IndexCoop(numeral(indexCoopValues.market_data.price_change_percentage_24h_in_currency.usd).format("0.00a") + "%");
+    setYUSDPrice(yusdPrice);
+    setDPIPrice(dpiPrice);
+    setWETHPrice(wethPrice);
+    setUMAPrice(umaPrice);
+    setIndexCoopPrice(indexCoopPrice);
+  }, [setYUSDPrice, setDPIPrice, setWETHPrice, setIndexCoopPrice, setUMAPrice]);
+
+  useEffect(() => {
+    if (status === "connected") {
+      fetchOnce();
+    }
+  }, [fetchOnce, status]);
+
+  const assetYUSD = (totalYUsdValue || 0) * (yusdPrice || 0);
+  const assetDPI = (totalDPIValue || 0) * (dpiPrice || 0);
+  const assetWETH = (totalWETHValue || 0) * (wethPrice || 0);
+  const assetUMA = (totalUMAValue || 0) * (umaPrice || 0);
+  const assetIndexBalance = (totalBalanceIndexCoop || 0) * (indexCoopPrice || 0);
+
+  const assets = [
+    {
+      name: "DefiPulse Index",
+      index: "DPI",
+      quantity: numeral(totalDPIValue).format("0,0.00"),
+      price: "$" + numeral(dpiPrice).format("0,0.00"),
+      change: change24DPI,
```

Comment:
if you go to the page, and scroll down immediately you will notice empty values on the table under the % change, adding pre defined empty values will solve that, for a better experience. 
```suggestion
      change: `(change24DPI ? change24DPI : "0.00%")`,
```